### DISCUSSION TITLE
Is there any public storage with all the maps ?

### DISCUSSION THREAD

**Discussion by leryss** [ppy/osu#13304] [Q&A] [answered]
Not sure if this is the right place to ask this question
I want to make a more complex map indexer, one which you can query for maps containing certain patterns (triples, streams, 1-2s etc) or statistics like % of sliders in the map etc. You get the idea.

I need a relatively large set of maps (atleast 10000 for starters), not sure if there is already a public storage with maps or if I should just use the API and download maps individually.

**Comment by peppy** [comment, marked as answer]
https://data.ppy.sh/2021_06_01_osu_files.tar.bz2 this should help you (see https://data.ppy.sh for all available data).

**Reply by leryss** [reply]
Thanks, this is exactly what I needed
